webrender_api/src/display_list.rs

```diff
@@ -915,18 +915,36 @@ impl DisplayListBuilder {
         self.save_state.take().expect("No save to clear in DisplayListBuilder");
     }
 
-    pub fn print_display_list(&mut self) {
+    /// Print the display items in the list to stderr. If the start parameter
```

Comment:
stdout is buffered often, so things written to stdout don't always show up if you crash right after, for example. This is one of the reasons why pretty much all of gecko logging goes to stderr, and if this goes to stdout it won't be interleaved properly with the rest of the output. Although maybe this should return a string rather than dumping it directly and the caller can decide where to put it.

(The rest of your comments make sense, will update the patch accordingly)

Reply:
FYI, my experience so far indicates that rust's `println!` flushes, but `print!` does not.

Reply:
Even if that's the case, these prints are going to be interleaved between prints from C++ code, so they should probably go to the same place.